service/host.go

```diff
@@ -164,6 +164,7 @@ func (s *manager) shutdown(err error, reason string, exitCode *int) (bool, error
 	}
 
 	// Flush tracing buffers
+	// GFM-378 uhttp.Client is not safe to send requests anymore
```

Comment:
This is not unique to http. If Stop is called, nothing is safe to call. Doesn't this happen to YARPC client too?